scripts/update_documents.py

```diff
@@ -0,0 +1,177 @@
+#! /usr/bin/env python3
```

Comment:
it seems a lot of this is the same as the previous script?

IIUC, it is because the previous one is for updating and putting them all in one place and this one is just about updating?

Reply:
Sorry, this is my mistake, I got my PRs mixed up. This script came earlier and was supposed to just update documents. I think this would be the script to keep for ongoing use , while the other one would be a one time migration that perhaps doesn't even need to be kept in the repo.